modules/disabling-etcd-encryption.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * authentication/encrypting-etcd.adoc
+
+[id="disabling-etcd_{context}"]
+= Disabling etcd encryption
+
+You can disable encryption of etcd data in your cluster.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Modify the API server object:
++
+----
+$ oc edit apiserver
+----
+
+. Set the `encryption` field type to `identity`:
++
+[source,yaml]
+----
+spec:
+  encryption:
+    type: identity
+----
+
+. Save the file to apply the changes.
++
+The decryption process starts. It may take 20 minutes or longer for this process to complete, depending on the size of your cluster.
+
+. Verify that etcd decryption was successful.
+
+.. Verify that the status of the `Encrypted` status condition is `DecryptionCompleted` for the OpenShift API server:
```

Comment:
I might say "Verify that the OpenShift API server decrypted:" here and consider adding more details after the command.